recipe/conda_build_config.yaml

```
@@ -4,3 +4,5 @@ c_stdlib:
 c_stdlib_version:              # [unix]
   - 2.17                       # [linux]
   - 11.0                       # [osx]
```

Not directly related, but this can be cleaned up as well (2.17 is the global default now, no need to respecify it).
```suggestion
c_stdlib_version:              # [osx]
  - 11.0                       # [osx]
```